sdk/tables/azure-data-tables/samples/README.md

```diff
@@ -48,6 +48,79 @@ pip install --pre azure-data-tables
 2. Set the environment variables specified in the sample file you wish to run.
 3. Follow the usage described in the file, e.g. `python sample_create_table.py`
 
+## Writing Filters
+
+### Supported Comparison Operators
+|**Operator**|**URI expression**|
+|------------|------------------|
+|`Equal`|`eq`|
+|`GreaterThan`|`gt`|
+|`GreaterTahnOrEqual`|`ge`|
+|`LessThan`|`lt`|
+|`LessThanOrEqual`|`le`|
+|`NotEqual`|`ne`|
+|`And`|`and`|
+|`Not`|`not`|
+|`Or`|`or`|
+
+### Example Filters
+
+#### Filter on `PartitionKey` and `RowKey`:
+```python
+parameters = {
+    "pk": PartitionKey,
+    "rk": RowKey
+}
+filter = "PartitionKey eq @pk and RowKey eq @rk"
+table_client.query_entities(filter=filter, parameter=pk)
+```
+
+#### Filter on Properties
+```python
+parameters = {
+    "first": first_name,
+    "last": last_name
+}
+filter = "FirstName eq @first or LastName eq @last"
+table_client.query_entities(filter=filter, parameter=pk)
+```
+
+#### Filter with string comparison operators
+```python
+filter = "LastName ge 'A' and LastName lt 'B'"
+table_client.query_entities(filter=filter)
+```
+
+#### Filter with numeric properties
+```python
+filter = "Age gt 30"
+table_client.query_entities(filter=filter)
+```
+
+```python
+filter = "AmountDue le 100.25"
+table_client.query_entities(filter=filter)
+```
+
+#### Filter with boolean properties
+```python
+filter = "IsActive eq true"
+table_client.query_entities(filter=filter)
+```
+
+#### Filter with DateTime properties
+```python
+filter = "CustomerSince eq datetime'2008-07-10T00:00:00Z'"
```

Comment:
When we add the bug fix for passing in `parameters` dictionary - we should add support for serializing datetime objects correctly like this. So you could do:
```python
parameters = { 'now': datetime.datetime.now()}
filter = "CustomerSince eq @now"
table_client.query_entities(filter=filter, parameters=parameters)
```